tools/release-create-tags.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+CURRENT_DIR=$(pwd)
+SETUP_FILENAME="setup.py"
+CONNECTOR_NAME_REGEXP="^google-datacatalog-([a-zA-Z]+)-connector$"
+
+release-create-tags::main(){
+
+    if [[ -z "${GITHUB_TOKEN}" ]]; then
+        echo "Undefined GITHUB_TOKEN variable"
+        release-create-tags::usage
+    fi
+
+    echo "START release tagging script"
+    echo ""
+
+    for connector_dir in `ls ${CURRENT_DIR} | grep connector`; do
+        echo " running on: ${connector_dir}"
+
+        setup_path=${CURRENT_DIR}/${connector_dir}/${SETUP_FILENAME}
+
+        release_version=$(sed -n "s/^ *version=['\'']//p" ${setup_path} | sed -n "s/['\'',]*$//p")
+
+        [[ ${connector_dir} =~ ${CONNECTOR_NAME_REGEXP} ]]
+        connector_simple_name="${BASH_REMATCH[1]}"
+
+        echo " tagging ${connector_simple_name} with release version: ${release_version}"
+
+        tag_name="${connector_simple_name}-${release_version}"
+
+        git tag ${tag_name}
+
+        if [[ $? -eq 0 ]]
+            then
+                echo " tag: ${tag_name} created"
+                git_remote_url="$(git remote get-url origin).git"
+                git_auth_remote_url="${git_remote_url/github.com/${GITHUB_TOKEN}@github.com}"
+                git push ${git_auth_remote_url} ${tag_name}
+                echo " tag: ${tag_name} pushed!"
+            else
+                echo " tag: ${tag_name} failed!"
+        fi
+
+        echo ""
+    done
+
+    echo 'END release tagging script'
+}
+
+release-create-tags::usage(){
+  echo 'SET: "GITHUB_TOKEN" environment variable'
```

Comment:
The script itself LGTM. Maybe it's worth elaborating a bit more on the `GITHUB_TOKEN" environment variable` description to let users know it means a personal access token. WDYT?

Reply:
I want to submit another PR with the `PyPI` publishing script and then document how they are executed, elaborating the required args, WDYT?

Reply:
Cool, that makes sense! Since you are planning this new documentation _layer_, I'm gonna add a few more comments to turn the script itself less tied to the release process and more focused on what it actually does: create tags according to the current version of each connector. 😃